element_lab/lab.py

```diff
@@ -77,18 +76,46 @@ class Protocol(dj.Lookup):
     protocol                : varchar(16)
     ---
     -> ProtocolType
-    protocol_description=''        : varchar(255)
+    protocol_description='' : varchar(255)
     """
 
-
 @schema
 class Project(dj.Lookup):
     definition = """
     project                 : varchar(32)
     ---
-    project_description=''         : varchar(1024)
+    project_description=''  : varchar(1024)
+    # Below included for archival export (e.g., NWB)
+    repositoryurl=''        : varchar(256) # URL to code for replication
+    repositoryname=''       : varchar(32)  # name of repository
+    pharmacology = ''       : varchar(2048) # Drugs used, how/when administered
+    viruses=''              : varchar(2048) # ID, source, date made, injection loc, volume
+    slices=''               : varchar(2048) # If slicing, preparation thickness, orientation, temperature, and bath solution
+    stimulus=''             : varchar(2048) # Generation method, how/when/where presented
```

Comment:
For the NWB export function, should we fetch this attribute from `element-sensory-stimulation`?

Reply:
That sounds good to me.  Could you please create a GitHub Issue in this repository to remind us?

Reply:
Agreed with Kabi, overall looks good. My major reservation here is I still don't think `pharmacology`, `viruses`, `slices`, `stimulus` should belong here in `Project`. 

The `Project` table is meant to serve as a "container" with description to house the many different sessions that belong in a particular project - e.g. use in conjunction with the `ProjectSession` table in the `element-session`. 

This should be generic, so meta information such as description, keywords, publication doi, etc. are  reasonable to have here, but `viruses`, `pharmarcology`, etc. are too specific to be here. (These can be part of other elements, or some user-defined upstream tables)